TodoListClient/TodoListClient.csproj

```diff
@@ -139,6 +136,23 @@
       <Generator>MSBuild:Compile</Generator>
     </Page>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights">
```

Comment:
Why do you need AppInsights?

Reply:
we don't :(
but this is a v1.0 sample, written by Vittorio, I believe. I'm not going to change it.